bin/docker-image-tool.sh

```diff
@@ -219,6 +227,8 @@ Options:
   -b arg                Build arg to build or push the image. For multiple build args, this option needs to
                         be used separately for each build arg.
 
+In dev the build enviroment copies in resource-managers/kubernetes/docker/src/main/dockerfiles
```

Comment:
Nit: does this need a little clarification ... does it mean the build environment copies something from A to B, or example copies are here?

Reply:
Yeah, it seems something is missing from this sentence.